session_4/blogspot/app/views/articles/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<br>
+<font color="#3385ff">
+    <h1>Editing Article</h1>
+</font>
+<hr>
+<br>
+<%= render 'form', article: @article %>
+<br>
+<br>
+<%= link_to 'Back', @article %> 
+<hr>
+<button type="button" class="btn btn-outline-danger">
```

Comment:
Alternatively, you can write this as:

```erb
<%= link_to 'Delete Article, @article, method: :delete, class: 'btn btn-outline-danger', data: {confirm: 'Are you sure?'} %>
```

That is, you don't need the `<button>` tag because of how styling works with bootstrap.